benchmarks/index.js

```diff
@@ -21,13 +21,13 @@ function printResults (results, n) {
 
 if (isMainThread) {
   const server = createServer((req, res) => {
-    setTimeout(() => {
+    process.nextTick(() => {
       res.end('payload')
-    }, 1)
+    })
   })
 
-  server.listen(0, () => {
-    const N = 10000
+  server.listen(() => {
+    const N = 1000
```

Comment:
so for some reason when I ran this with 10000 requests, my localhost server would timeout after the undici-fetch execution finishes. Would someone try pulling this branch locally, changing this to `10000` and running `npm run bench` to see what happens?